products/cloudrun/api.yaml

```diff
@@ -219,6 +219,11 @@ objects:
   name: Service
   kind: Service
   base_url: projects/{{project}}/locations/{{location}}/services
+  async: !ruby/object:Api::PollAsync
+    actions: ['create', 'update']
+    operation: !ruby/object:Api::PollAsync::Operation
+      # TODO chrisst - update to cloudrun v1 url structure once v1 PR is merged.
+      full_url: 'projects/{{project}}/locations/{{location}}/services/{{name}}'
```

Comment:
If I understand this PR correctly, this URL isn't used in templates anywhere. 

It seems like PollAsync is really more about specifying custom async code than configuring async logic (at the moment)

Reply:
ah yeah, that was a hold over before moving to the GET url by default.